src/current/_includes/releases/v23.1/v23.1.5.md

```diff
@@ -0,0 +1,93 @@
+## v23.1.5
+
+Release Date: July 5, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-5-security-updates">Security updates</h3>
+
+- The full set of TLS ciphers that was present in v22.1 have ben included in the old cipher suites list, which can be enabled with the `COCKROACH_TLS_ENABLE_OLD_CIPHER_SUITES` environment variable. [#105370][#105370]
+
+<h3 id="v23-1-5-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Added new [cluster settings](../v23.1/cluster-settings.html) in the `server.oidc_authentication.generate_cluster_sso_token` namespace that support using OIDC to generate a [JWT auth token for cluster SSO](../v23.1/sso-sql.html). [#105643][#105643]
+
+<h3 id="v23-1-5-sql-language-changes">SQL language changes</h3>
+
+- Improved the inline documentation and error messages related to [`SHOW RANGES`](../v23.1/show-ranges.html). [#105278][#105278]
+- Allow inbound foreign keys on TTL tables. [#105341][#105341]
```

Comment:
The present tense on this one feels a little off among the other notes.

```suggestion
- Allowed inbound foreign keys on TTL tables. [#105341][#105341]
```

Or, could do something like "Inbound foreign keys on TTL tables are now allowed." 
If I have the meaning correct!

Reply:
Edited